client/scss/components/_keyboard-shortcuts.scss

```diff
@@ -0,0 +1,32 @@
+.keyboard_shortcut {
+  width: 100%;
+
+  &__category {
+    font-size: 20px;
+    display: flex;
+    flex-direction: row;
+    font-weight: 400;
+    margin-bottom: 0.75rem;
+    margin-top: 1.2rem;
+  }
```

Comment:
Done; as I have removed nesting, I gave two new class names to label and key th & td because id nesting is removed. This might affect other th td in the DOM.

Reply:
It should be fine, as the classname is unique to the ones in this new template only.